### PR TITLE
[PLATFORM-1473] Support opentelemetry 0.21

### PR DESCRIPTION
**Pull request by MaeIsBad**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-1473/Support-opentelemetry-0.21

### CODE REVIEW

**Review by cpiemontese — COMMENTED** [review]
I like the approach, because it gives us a way to release the latest version of the otel deps but lets users rollback on their own if it doesn't work

I think we should limit the supported versions and not end up like the actix library, but it's a matter of "discipline"

Is there a `README` to update detailing this a bit? I.e. that we support two version and they can be used like so and blah blah

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,15 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
----
-
 ### Added
 
 - Support for opentelemetry 0.21
+- `tracing_opentelemetry_0_21` and `tracing_opentelemetry_0_21` features
```
Comment (cpiemontese) [inline CHANGELOG.md:13]: I don't understand the difference :thinking:

**Review by cpiemontese — DISMISSED** [review]
One comment but I may be dumb

File: CHANGELOG.md (hunk 1 above)
Comment (MaeIsBad) [reply CHANGELOG.md]: Oops my bad, fixed

**Review by cpiemontese — DISMISSED** [review]

**Review by cpiemontese — APPROVED** [review]

**Pull request merged**